docs/data/quick-start.md

```diff
@@ -0,0 +1,791 @@
+    }});
+    ```
+
+=== "iOS"
+
+    ```obj-c title="Objective-C"
+    [[Amplitude instance] logEvent:@"Button Clicked"];
+    ```
+
+    ```swift title="Swift"
+    Amplitude.instance().logEvent("Button Click")
+    ```
+
+=== "Java"
+
+    ```java
+
+    amplitude.logEvent(new Event("Button Clicked", "test_user_id"));
+    ```
+
+=== "Python"
+
+    ```python title="Python"
+    from amplitude import BaseEvent
+
+    amplitude.track(
+        BaseEvent(
+            event_type="type of event",
+            user_id="USER_ID",
+            device_id="DEVICE_ID",
+            event_properties={
+                "source": "notification"
+            }
+    ))
+    ```
+
+=== "React Native"
+
+    ```ts title="TypeScript"
+
+    import { track } from '@amplitude/analytics-react-native';
+
+    const eventProperties = {
+        buttonColor: 'primary',
+    };
+    track('Button Clicked', eventProperties);
+    ```
+
+=== "Flutter"
+
+    ```dart title="Dart"
+
+    amplitude.logEvent('BUTTON_CLICKED', {"Hover Time": "100ms"});
+    ```
+
+=== "Unity"
+
+    ```c# title="C#"
+    import 'package:amplitude_flutter/amplitude.dart';
+
+    amplitude.logEvent('MyApp startup', eventProperties: {
+      'friend_num': 10,
+      'is_heavy_user': true
+    });
+    ```
+
+=== "Unreal"
+
+    ```c++ title="C++"
+
+    TArray<FAnalyticsEventAttribute> AppendedAttributes;
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key1"), TEXT("Test Event value1"));
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key2"), TEXT("Test Event value2"));
+    FAnalytics::Get().GetDefaultConfiguredProvider()->RecordEvent(TEXT("Game Started"), AppendedAttributes);
+    ```
+
+=== "GO"
+
+    ```go title="GO"
+
+    amplitude.Track(amplitude.Event{
+           EventType: "Button Clicked",
+           EventOptions: amplitude.EventOptions{
+           UserID:   "user-id",
+           DeviceID: "device-id",
+        },
+        EventProperties: map[string]interface{}{"source": "notification"},
+    })
+
+    ```
+
+## 3. Check for success
+#### Ingestion Debugger
+    1. Login to you Amplitude Analytics Orgnization.
+    2. Click the `Data Source tab` at the bottom of the navigation bar
+    3. Choose the `Ingestion Debugger` section. You can check the Requests or which User or Device has been throttled.
+
+Learn more about [Ingestion Debugger](https://help.amplitude.com/hc/en-us/articles/360044835531).
+#### User Lookup
+
+After you fired an event, you can navigate to the `User & Account Look-Up` tab in Amplitude validate your event.
+
+Learn more about [User Lookup](https://www.docs.developers.amplitude.com/data/debugger/#user-lookup).
+
+#### Event Explorer
+In any Amplitude chart, click `Select events using your product`. You are able to watch the events roll in, in near-real time.
+
+Learn more about [Event Explorer](https://help.amplitude.com/hc/en-us/articles/360050836071-Event-Explorer-View-event-streams-in-real-time).
+#### Instrument Explorer
```

Comment:
Minor, should we mention that this option is for browser SDK only specifically?

Reply:
I think we should only show this for browser SDKs

Reply:
@caseyamp  Is it possible to add some `if else` statement and check what the current tab is

Reply:
Sorry for the delay! @yuhao900914 not without template changes and extensions -- it would be a non-trivial change. If you only want to show content on one tab, include the content on the tab itself. Tabs can include any markdown (and arbitrary HTML, I think)

If you want to show the content on several tabs, you can use a Snippet. Ping me in Slack if you want to explore that option.

Reply:
We cannot show different content based on the current tab for the contents out of the tab. I will add `Browser SDKs Only` in the !!note as the workaround.